src/extractor/guidance/intersection_handler.cpp

```diff
@@ -375,19 +391,29 @@ std::size_t IntersectionHandler::findObviousTurn(const EdgeID via_edge,
 
     const EdgeData &in_data = node_based_graph.GetEdgeData(via_edge);
     const auto in_classification = in_data.road_classification;
+
+    const auto obvious_by_road_class = [](const RoadClassification in_classification,
+                                          const RoadClassification obvious_candidate,
+                                          const RoadClassification compare_candidate) {
+        const bool has_high_priority =
+            2 * obvious_candidate.GetPriority() < compare_candidate.GetPriority();
```

Comment:
This assumes a certain structure in road priorities. Can we make `2` a constant and document this relationship in the road priority related code?